### ISSUE TITLE
What transports & pipelines we should include?

### ISSUE THREAD

**Issue by H4ad** [H4ad/nodejs-logging-proposal#1]
By default pino will output to `stdout` using `SonicBoom` at https://github.com/pinojs/pino/blob/main/lib/tools.js#L330-L336

Should we include more transports, like:

- file: pino supports file by default: https://github.com/pinojs/pino/blob/main/docs/transports.md#pinofile
- pretty: https://github.com/pinojs/pino-pretty
- http: https://www.npmjs.com/package/pino-http-send
- websocket: https://www.npmjs.com/package/@abeai/pino-websocket
- socket: https://www.npmjs.com/package/pino-socket

Should we include pipelines like:

- syslog: https://www.npmjs.com/package/pino-syslog
- logfmt: https://github.com/botflux/pino-logfmt

Also, the way to expose those built-in transports can be something like:

```js
setOptions({
  transport: {
    target: 'node:logging/transports/pretty',
  },
})
```

Basically we can group/expose the transports via `node:logging/transports/<name>` and the pipelines via `node:logging/pipelines/<name>`.
